asset-tracking-history-saver/src/index.ts

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env node
+import * as Ably from 'ably/promises';
+import { Command } from 'commander';
+import { CHANNEL_PREFIX, CLIENT_ID } from './consts';
+import { getEnhancedLocationsFromChannelHistory } from './getDataFromChannelHistory';
+import { getDataFromFile } from './getDataFromFile';
+import { saveDataToFile } from './saveDataToFile';
+import { AblyCredentialsFileData, ConfigurationFileData, EnhancedLocationUpdate } from './types';
+const onProcessInterrupted = require('death');
```

Comment:
Not sure why you're mixing CommonJS and ESM here. I would assume you could just import from 'death' as normal and use the `resolveJsonModule` tsc compiler option to do the same for package.json

Reply:
I'm not sure why I did that either :sweat_smile: I've fixed the `death` import https://github.com/ably/ably-asset-tracking-common/pull/43/commits/12986cb8ed3bb765bb011701e2b10131ce3f5b0b but the `package.json` thing is a bit more complicated.

If I simply add the `resolveJsonModule` then the compiler complains that the `package.json` is not under the root directory of the project :confused: I've found [this SO answer](https://stackoverflow.com/a/61467483) that shows how to fix it but I wonder if that's something that we want to do for such a small script :wink: Please let me know how should we proceed with that :bow:

Reply:
Ah, right I didn't notice you were using `rootDir`, yeah it probably doesn't matter too much so I'll resolve this thread and leave it to you to decide 👍